helpers.go

```diff
@@ -0,0 +1,35 @@
+package validate
+
+import (
+	"reflect"
+)
+
+// RequireField returns a error for the given field if provided value is zero.
+func RequireField(field string, value interface{}) error {
```

Comment:
@geoah the implementation of this helper is probably the area I have least confidence in having done things in a non-stupid way. Though there's a possibility there's stupid in other places too... lol